cmd/maestro/migrate/cmd.go

```diff
@@ -24,18 +23,17 @@ func NewMigrationCommand() *cobra.Command {
 	}
 
 	dbConfig.AddFlags(cmd.PersistentFlags())
-	cmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
```

Comment:
why do you remove it? We do not need it anymore?

Reply:
the flags is added into root command, so that all sub comands can share it.